mgmt_cli_test.py

```diff
@@ -196,6 +196,23 @@ def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspac
         self.restore_backup_from_backup_task(mgr_cluster=mgr_cluster, backup_task=backup_task,
                                              keyspace_and_table_list=per_keyspace_tables_dict)
 
+    def restore_schema_with_task(self, mgr_cluster, backup_task, timeout):
+        snapshot_tag = backup_task.get_snapshot_tag()
+        restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=self.locations,
+                                                       snapshot_tag=snapshot_tag)
+        restore_task.wait_and_get_final_status(step=30, timeout=timeout)
+        assert restore_task.status == TaskStatus.DONE, f"Schema restoration of {snapshot_tag} has failed!"
+        self.db_cluster.restart_scylla()  # After schema restoration, you should restart the nodes
+
+    def restore_data_with_task(self, mgr_cluster, backup_task, timeout):
+        snapshot_tag = backup_task.get_snapshot_tag()
+        restore_task = mgr_cluster.create_restore_task(restore_data=True, location_list=self.locations,
+                                                       snapshot_tag=snapshot_tag)
+        restore_task.wait_and_get_final_status(step=30, timeout=timeout)
+        assert restore_task.status == TaskStatus.DONE, f"Schema restoration of {snapshot_tag} has failed!"
```

Comment:
The assert message here is wrong, here we don't restore the schema at all....

Reply:
Created a pr to amend that: https://github.com/scylladb/scylla-cluster-tests/pull/5856